### PR TITLE
INFODEV-230 [MVP] Rackspace Migration Manager: How-To article "What to expect during migration

### PR DESCRIPTION
**Pull request by cyrichardson**
Add how to article requested by support team using content from SME.

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-28aad57250/how-to/). :bow:

**Comment by cyrichardson** [comment]
@kmsholcomb This is a new one that will need your technical editing touch. I will create a card in our JIRA board and create a task for you. Thank you!

File: content/rackspace-migration-manager/what-to-expect-during-migration.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+**Timeline**
+
+The migration timeline depends on the size of the instances that are being
+migrated, network speeds, and the amount of data (for example, a system disk or
+CBS volumes).
+
+**Downtime**
+
+If the user chooses to use *live migrate* (which is coming soon), no downtime
```
Comment (amitgandhinz) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: remove the coming soon part.  this will be in the UI before we launch.

Comment (amitgandhinz) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: may also want to add the following disclaimer around the 'live migrate' option:

```When enabled, the Rackspace Migration Manager will not shut down the instance before migrating the image.  When this option is used, file system integrity on the migrated image cannot be guaranteed.```

File: content/rackspace-migration-manager/what-to-expect-during-migration.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+**Costs and fees**
+
+There is a cost for the support-vm that is spun up for each server that is
+being migrated. The customer is also responsible for transfer bandwidth to AWS
+during a migration. After migration, there might be resources in AWS or
+Rackspace Public Cloud that customer will be paying for. It is the customer's
+responsibility to remove any resources that they do not need. Temporary
+resources created by the migration might not get cleaned up. If this occurs,
+the customer might need to clean up these resources to avoid the costs.
+
```
Comment (amitgandhinz) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: can we make this more _nicer_ for the customer facing language.

**Comment by kmsholcomb** [comment]
Thanks, @cyrichardson. I'll get started on it now, since I will out on PTO until Thursday.

**Comment by cyrichardson** [comment]
@kmsholcomb Thank YOU! It's a short one.

File: content/rackspace-migration-manager/what-to-expect-during-migration.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+**Responsibilities**
+
+The terms and conditions indicate that Rackspace is not responsible for the
```
Comment (amitgandhinz) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: I talk about the terms and conditions here, based on discussions with Trent, but I'm not sure where they are spelled out (the UI is only linking to the FAWS T&C and the Global Agreements).  We need to find this and link to it here, then add the summarization of the responsibilities.

File: content/rackspace-migration-manager/what-to-expect-during-migration.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+migration. The customer must verify that all data was migrated. The customer is
+responsible for all costs, and must check the security of the resources that
+have been migrated. The customer is also responsible for updating application
+and DNS configurations to use any new IPs on the AWS side.
+
+**Scheduling a migration**
+
+Scheduling a migration can be done in the Control Panel.
```
Comment (amitgandhinz) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: Maybe link to the How To article for it to guide them though scheduling a migration.

File: content/rackspace-migration-manager/what-to-expect-during-migration.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+**Different migration types**
+
+(Need content here. Amit was not sure what was meant here. I don't think
```
Comment (amitgandhinz) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: If this means Lift and Shift, then we should explain to the customer what that means, and that we won't be rearchitecting their app for the cloud.

File: content/rackspace-migration-manager/what-to-expect-during-migration.md (hunk 1 above)
Comment (cyrichardson) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: Both items above done.

File: content/rackspace-migration-manager/what-to-expect-during-migration.md (hunk 2 above)
Comment (cyrichardson) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: I tried to make it nicer.

File: content/rackspace-migration-manager/what-to-expect-during-migration.md (hunk 3 above)
Comment (cyrichardson) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: If someone can provide the link to terms and conditions, I'll add it.

File: content/rackspace-migration-manager/what-to-expect-during-migration.md (hunk 4 above)
Comment (cyrichardson) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: When the How-To article exists, we can add the link. The content for this article has yet to be supplied.

File: content/rackspace-migration-manager/what-to-expect-during-migration.md (hunk 5 above)
Comment (cyrichardson) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: Done

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-975965205d/how-to/)

**Comment by kmsholcomb** [comment]
Hi @cyrichardson. Edits are attached. I had some questions and suggestions, and some markup. I am heading out for PTO now; I'll be back Thursday. If something doesn't make sense to you, just ignore it! :) 

[What to expect during migration to AWS_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1208423/What.to.expect.during.migration.to.AWS_editsKH.docx)
[What to expect during migration to AWS_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1208424/What.to.expect.during.migration.to.AWS_editsKH.pdf)

**Comment by cyrichardson** [comment]
Thanks, @kmsholcomb!

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c417ff54b6/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-2181865bc5/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-65fd13efaf/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-895fc2091c/how-to/)

**Comment by cyrichardson** [comment]
Note that the link to the RMM terms and conditions is also in the UI. Here are the links that Armando provided - I added the last one to this article because it is specific to RMM:

https://www.rackspace.com/information/legal/GSA
https://www.rackspace.com/information/legal/awssupport
https://www.rackspace.com/information/legal/migrationmanagertool

**Comment by cyrichardson** [comment]
@amitgandhinz Please recheck when you have time. The only thing I was not able to do is add a link to the control panel navigation article - it does not yet exist.

**Comment by amitgandhinz** [comment]
LGTM.  You may want to get feedback from Larry Hau also since they requested these.

**Comment by cyrichardson** [comment]
Thanks, @amitgandhinz! Will do.

**Comment by cyrichardson** [comment]
@amitgandhinz When you have time, would you please add Larry as a member so I can @ him and add him as a reviewer?

**Comment by amitgandhinz** [comment]
@th3incredibl3hulk  <- larry hau (lol)

**Comment by cyrichardson** [comment]
@th3incredibl3hulk Please review this new How-To article when you have time. You can go to Files changed at the top of this page, then hover over the plus sign (+) on the left side of the text of the article to add any review comments. Thanks!

File: content/rackspace-migration-manager/what-to-expect-during-migration.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+There is a cost for the support VM that is created for each server that is
+being migrated.
+
+You are also responsible for the cost of the transfer bandwidth to AWS during
+a migration.
+
+After migration, there might be resources in AWS or Rackspace Public Cloud
+that you will be expected to pay for. It is your responsibility to remove any
+resources that you do not need.
+
```
Comment (McNinja) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: There WILL be an additional cost, the customer will be paying for both RAX & AWS till they clean up one side. This needs to be painfully clear that this is on the customer to clean up the RAX side or use a combination, no credits will be handed out due to them forgetting about resources on 1 side.

File: content/rackspace-migration-manager/what-to-expect-during-migration.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+The migration timeline depends on the size of the instances that are being
+migrated, network speeds, and the amount of data (for example, a system disk or
+CBS volumes).
+
+### Downtime
+
+If you choose to use *live migrate*, no downtime of the Rackspace Public
+Cloud is expected. Otherwise, the instance will be down for about 10 to 20
+minutes while the disk is cloned.
+
```
Comment (McNinja) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: 10 - 20 minutes is way to optimistic, in our testing base images with no data on them took about this timne. I would expect this to be 2-10 times with production instances

File: content/rackspace-migration-manager/what-to-expect-during-migration.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+### Responsibilities
+
+The [terms and conditions](https://www.rackspace.com/information/legal/migrationmanagertool)
+indicate that Rackspace is not responsible for the
+migration. You must verify that all data was migrated. You are also
+responsible for all costs, and must check the security of the resources that
+have been migrated. Additionally, you are responsible for updating application
+and DNS configurations to use any new IP addresses in AWS.
+
```
Comment (McNinja) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: I would like to call out that Rackspace will NOT perform OS level actions to get their application/server working on AWS. Most if not all servers will need to have work done due to the IP changes that are occurring. This is 100% on the customer and Rackspace support will not log in.

File: content/rackspace-migration-manager/what-to-expect-during-migration.md (hunk 6 above)
Comment (cyrichardson) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: Make changes, but please review them.

File: content/rackspace-migration-manager/what-to-expect-during-migration.md (hunk 7 above)
Comment (cyrichardson) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: Changes made that will need review.

File: content/rackspace-migration-manager/what-to-expect-during-migration.md (hunk 8 above)
Comment (cyrichardson) [inline content/rackspace-migration-manager/what-to-expect-during-migration.md]: Changes made that need to be reviewed.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a866a200df/how-to/)

**Comment by cyrichardson** [comment]
@McNinja @sameerkotasthane Please review the changes that I made bases on @McNinja's review comments. Thanks!

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-55d658d657/how-to/). :bow:

**Comment by cyrichardson** [comment]
Closing in favor of https://jira.rax.io/browse/INFODEV-424

**Pull request closed without merging**